test/system/tests/clearlock.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#;**********************************************************************;
+#
+# Copyright (c) 2017, Intel Corporation
```

Comment:
Unless you are working on behalf of Intel, I don't think this copyright notice is correct.